resources/remix.md

```diff
@@ -0,0 +1,135 @@
+ 
+## Document
+
+This document should contains:
+  - General description of the project.
+  - Contact of involved people.
+  - Github repository where the project is stored.
+  - Description about how to make it work.
+  - What is left to be done in order to be usable in production.
+  - What were the main problems you encountered while developing and how did you solve them.
+  - What should have been done differently -if that applies.
+  - How would it be possible to extend contribution beyond the current project.
+    
+We really recommend to take time for that if you want to apply for a sponsorship.
+
+## How to get started
+
+Once you have choosen a thema/plugin you have interest on:
+ - Read carefully this document.
+ - Join https://gitter.im/ethereum/remix-dev-plugin
+ - Come to speak to us either online or directly at the event
+ - Post a message in the issue saying that you are starting it with your name, the name of your team (super important!), your contact (like mail address and/or git contact)
+ - Feel free to come to us if you have any Ethereum related question or anything you want to talk about ;)
+ - Close to the end of the hackathon, Please send us a document describing your work (see the previous section). We will use it for selecting max 4 team that will receive devgrant from the Ethereum Foundation.
+ - After the event we really like to keep contact with for further contribution, collaboration, ...
+
+
+## Setting up the development environement
+
+Here is a quick description that will help you to setup a dev environment:
+The plugin API uses iframe to talk to Remix. 
+https://github.com/ethereum/remix/blob/master/docs/remix_plugin.md
+https://github.com/ethereum/remix/blob/master/docs/remix_plugin_api.md
+
+
+We explain here how to get `remix-plugin`. A package that abstracts the `iframe` layer in order to make the development easier.
+
+### easy way: cloning the sample project and setting up the plugin
+
+follow the README of https://github.com/yann300/remix-plugin
+
+### manual setup: building the Remix API lib and setting up the plugin
+
+git clone https://github.com/ethereum/remix-ide
+npm install
+cd src/app/plugin
+
+npm install
+npm run browserify
+This will generate the file `bundle.js`. Reference this file in any html page to start using the API.
+
+Once you have this `bundle.js` (or `require('remix-plugin')`)
+
+Create a folder (not necessarily in Remix) where the plugin code should go.
+copy `bundle.js` to it.
+Go in `test-browser/plugin` from the remix-ide project and copy the file `index.html` and `plugin.js` to your working directory.
+Serve this folder through a web server `http-server .`
+
+Go in Remix / settings tab, under the `Plugin` section paste the following declaration:
+
+{
+    "title": "<name of plugin>",
+    "url": "http://127.0.0.1:<port>"
+}
+
+Then start the plugin by licking on its icon.
```

Comment:
licking ?-) 
Nice typo! I think you mean clicking ;-)
and please delete the blank lines afterwards